package.json

```diff
@@ -121,6 +121,7 @@
   "dependencies": {
     "@paulcbetts/system-idle-time": "1.0.4",
     "i18next": "11.2.3",
-    "i18next-node-fs-backend": "1.0.0"
+    "i18next-node-fs-backend": "1.0.0",
+    "suncalc": "^1.8.0"
```

Comment:
this will need to be in `app/package.json` as well so it get's packaged to the installers and so on 

cc @aleuca @Sveijk